libs/verifiable-credentials/src/main/java/uk/gov/di/ipv/core/library/verifiablecredential/helpers/VcHelper.java

```diff
@@ -223,4 +224,13 @@ private static Integer getAge(String dobValue) {
             return null;
         }
     }
+
+    public static Vot getVcVot(SignedJWT vc) throws UnrecognisedVotException {
+        try {
+            String vot = vc.getJWTClaimsSet().getStringClaim(VOT_CLAIM_NAME);
+            return vot == null ? null : Vot.valueOf(vot);
+        } catch (ParseException | IllegalArgumentException e) {
+            throw new UnrecognisedVotException("Invalid VOT found for this VC");
```

Comment:
Seems reasonable

Reply:
On this, since the ParseException would happen in `getStringClaim` || `getJWTClaimsSet` before its able to get a vot this would mean the vot value would not be available in some cases thereby -- correct me if I am wrong (Java newbie) -- transferring the problem into the catch block. 

Potential solution could be having to separate catch blocks. And since it's a very unlikely error, is it still worth doing that?

Reply:
Ah yeah, that's a good point. We may not actually have a vot. 

I think you're right. This is very unliely, and if we ever did need to investigate we could fetch the VC in question and work out what happened.

Reply:
I think in that case we may want to change the rethrown exception